README.md

```diff
@@ -148,14 +181,14 @@ Write ruby code that will create a person.
 
 Your answer:
 ```
-Replace this with your answer
+dude = Person.create(id: 1, name: "Sam", age: 26)
```

Comment:
I don't think you want to include an `id` it should automatically assign one when you create a person.